db/seeds.rb

```diff
@@ -17,17 +17,21 @@
 end
 
 (1..10).each do |i|
-    value = 11-i
-    start_time = Time.now + rand(3.days)
-    t_price = (20..30).to_a.sample
-    Seance.find_or_create_by(movie_id: i, hall_id: value, :starts_at => start_time, price: t_price)
+  value = 11-i
+  start_time = Time.now + rand(3.days)
+  t_price = (20..30).to_a.sample
+  Seance.find_or_create_by(movie_id: i, hall_id: value, :starts_at => start_time, price: t_price)
 end
 
 (3..10).each do |i|
-    value = (i-1)*5
-    Promotion.find_or_create_by(value: i, information: "#{value}% discount for snack set in snack bar")
-  end
+  value = (i-1)*5
+  Promotion.find_or_create_by(value: i, information: "#{value}% discount for snack set in snack bar")
+end
 
 (1..100).each do |i|
```

Comment:
the number used here is irrelevant so it may be easier or more explicit to write
100.times do 
we then know there are 100